spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -534,7 +534,7 @@ private static void importUnpartitionedSparkTable(
                 .createDataset(Lists.transform(files, f -> f.path().toString()), Encoders.STRING())
                 .toDF("file_path");
         Dataset<Row> existingFiles =
-            loadMetadataTable(spark, targetTable, MetadataTableType.ENTRIES);
+            loadMetadataTable(spark, targetTable, MetadataTableType.ENTRIES).filter("status!=2");
```

Comment:
@dramaticlly sorry for all the nit pick comments, but can we do "status != 2" for readability (spaces between).

Ref: Javadoc for Dataset.filter():   https://spark.apache.org/docs/2.4.1/api/java/org/apache/spark/sql/Dataset.html#filter-org.apache.spark.sql.Column-

Reply:
thank you @szehon-ho , added space in between and rebased